frontend/javascripts/types/schemas/dataset_view_configuration_defaults.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import {
+  getDefaultLayerViewConfiguration,
+  defaultDatasetViewConfiguration,
+} from "types/schemas/dataset_view_configuration.schema";
+import { type APIDataset, type APIDataLayer } from "types/api_flow_types";
+import { getDefaultIntensityRangeOfLayer } from "oxalis/model/accessors/dataset_accessor";
+import { validateObjectWithType } from "types/validation";
+
+const eliminateErrors = (instance: Object, errors: Array<*>, defaults: Object) => {
+  errors.forEach(error => {
+    if (error.name === "required") {
+      instance[error.argument] = defaults[error.argument];
+    } else if (error.name === "additionalProperties") {
+      delete instance[error.argument];
```

Comment:
Does `error.argument` always refer to a top-level property or can it be nested?

Reply:
It always refers to a top-level property.